frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx

```diff
@@ -63,9 +63,11 @@ class BundleDetail extends React.Component<
                 this.fetchBundleContents();
                 return;
             }
-            if (this.state.bundleInfo.bundle_type === 'run' && (this.state.bundleInfo.state === 'running' 
-                || this.state.bundleInfo.state === 'preparing' || this.state.bundleInfo.state === 'starting'
-                || this.state.bundleInfo.state === 'staged')){
+            if (this.state.bundleInfo.bundle_type === 'run' && (
+                    this.state.bundleInfo.state === 'created' 

```

Comment:
I think we should check for more bundle states -- in fact, use a regex `(uploading|created|staged|making|starting|preparing|running)` (comes from https://github.com/codalab/codalab-worksheets/blob/6e0d3e9faff013db8d77125329649dfec02511df/scripts/create_sample_worksheet.py#L38)